### DISCUSSION TITLE
How could use NICU data?

### DISCUSSION THREAD

**Discussion by sujungleeml** [MIT-LCP/mimic-code#1377] [MIMIC-IV] [answered]
I cannot extract NICU data from icustays on MIMIC 4, unlike MIMIC 3.
However, I found NICU patients through the careunit in the transfers table, and these patients seems to be enrolled in the database. In this table, 29,363 rows of nicu history were found. (admit, tansfer included)

I investigated the BP and HR itemids to check the vital signs of these patients, but the vital signs of neonatal patients could not find in chartevents.

Is the NICU data intentionally not provided?

**Comment by marymlucas** [comment, marked as answer]
Hi @sujungleeml 
In v2 of MIMIC-IV the data for neonates was removed, which is probably why you can't find it.  If you read the **Major Changes** section at [https://physionet.org/content/mimiciv/2.0/](https://physionet.org/content/mimiciv/2.0/) it states:
_Neonates have been removed from the dataset. Neonatal data will be released in a separate project with data from the neonatal intensive care unit._

Hope this helps
Mary

**Reply by sujungleeml** [reply]
Thanks for your kind reply. If so, is there not yet an upcoming schedule or reference page for a separate neonate intensive care unit dataset?

**Reply by marymlucas** [reply]
Not that I've seen.
